wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSettings.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CDS\Modules\Site;
+
+use InvalidArgumentException;
+
+class SiteSettings
+{
+    protected string $admin_page = 'cds_site_setings';
+
+    public function __construct()
+    {
+    }
+
+    public static function register()
+    {
+        $instance = new self();
+
+        add_action('admin_menu', [$instance, 'collectionSettingsAddPluginPage']);
+        add_action('admin_init', [$instance, 'collectionSettingsPageInit']);
+
+        add_filter('collection_settings_option_group', function ($capability) {
+            return user_can('manage_options');
+        });
+    }
+
+    public function collectionSettingsAddPluginPage()
+    {
+        add_options_page(
+            __('Collection Settings'), // page_title
+            __('Collection Settings'), // menu_title
+            'manage_options', // capability
+            'collection-settings', // menu_slug
+            array( $this, 'collectionSettingsCreateAdminPage' ) // function
+        );
+    }
+
+    public function collectionSettingsCreateAdminPage()
+    {
+
+        ?>
+
+        <div class="wrap">
+            <h1><?php _e('Collection Settings', 'cds-snc') ?></h1>
+            <p></p>
```

Comment:
Empty paragraph, we can probably lose this.

Reply:
removed